### PR TITLE
Update phyrexian syntax, add ConniveAI

### PR DESCRIPTION
**Pull request by kevlahnota**
- fix mobile display for compleated symbols
- fix Raffine, Scheming Seer

### CODE REVIEW

**Comment by Hanmac** [comment]
Comp Rules say:
> the hybrid Phyrexian symbols {W/U/P}, {W/B/P}, {U/B/P}, {U/R/P}, {B/R/P}, {B/G/P}, {R/G/P}, {R/W/P}, {G/W/P}, and {G/U/P}

should we try to update it that way?

**Comment by kevlahnota** [comment]
> Comp Rules say:
> 
> > the hybrid Phyrexian symbols {W/U/P}, {W/B/P}, {U/B/P}, {U/R/P}, {B/R/P}, {B/G/P}, {R/G/P}, {R/W/P}, {G/W/P}, and {G/U/P}
> 
> should we try to update it that way?

for consistency reason, yes

**Comment by Hanmac** [comment]
there is the Rule:
> 107.4f Phyrexian mana symbols are colored mana symbols: {W/P} is white, {U/P} is blue, {B/P} is black, {R/P} is red, and {G/P} is green. A Phyrexian mana symbol represents a cost that can be paid either with one mana of its color or by paying 2 life. There are also ten hybrid Phyrexian mana symbols. A hybrid Phyrexian mana symbol represents a cost that can be paid with one mana of either of its component colors or by paying 2 life. A hybrid Phyrexian mana symbol is both of its component colors.
Example: {W/P}{W/P} can be paid by spending {W}{W}, by spending {W} and paying 2 life, or by paying 4 life.

This should probably be updated in the Desktop and Mobile Ports?

**Comment by Hanmac** [comment]
because you updated the ManaCostShard you should probably update all cardscripts using Phyrexian Mana too

**Comment by kevlahnota** [comment]
> because you updated the ManaCostShard you should probably update all cardscripts using Phyrexian Mana too

I think it was handled by updateAbilityManaSymbols on CardReaderExperiments, it's working without updating the old scripts

**Comment by Hanmac** [comment]
because https://github.com/Card-Forge/forge/blob/8283720e4ae44c8bd2b7e0f54da54de327a2e344/forge-core/src/main/java/forge/card/mana/ManaCostShard.java#L199-L226 checks for Flags so PWG WGP or WPG or what ever is the same for the ManaCostShard

File: forge-ai/src/main/java/forge/ai/ability/ConniveAi.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import forge.game.spellability.SpellAbility;
+import forge.game.zone.ZoneType;
+
+public class ConniveAi extends SpellAbilityAi {
+    @Override
+    protected boolean doTriggerAINoCost(Player ai, SpellAbility sa, boolean mandatory) {
+        final Card source = sa.getHostCard();
+        boolean preferred = true;
+        CardCollection list;
+        final int amount = AbilityUtils.calculateAmount(source, sa.getParamOrDefault("ConniveNum", "1"), sa);
```
Comment (Hanmac) [inline forge-ai/src/main/java/forge/ai/ability/ConniveAi.java:20]: amount not used ?

Comment (kevlahnota) [reply forge-ai/src/main/java/forge/ai/ability/ConniveAi.java]: yeah, I think the discard part is already handled.

**Comment by Hanmac** [comment]
I'm surprised that Connive doesn't have more AI Logic

@Agetian 
`X target Creatures Connive 1` like `Change of Plans` probably need AI logic to not choose X > Library Size (+ Buffer)

**Review by Agetian — APPROVED** [review]

**Pull request merged**